src/elf/mod.rs

```diff
@@ -202,22 +202,45 @@ if_sylvan! {
         pub fn is_object_file(&self) -> bool {
             self.header.e_type == header::ET_REL
         }
+
+        pub fn parse_hdr(bytes: &'a [u8]) -> error::Result<Header> {
```

Comment:
One last thing: we need doc comment on this function just for consistency, and i think it should be named `parse_header`; the type is `Header`, and the field is `header` so it is unusual to use the `elf.h` spelling here (but nowhere else).

Reply:
Got it. Done.